images/win/scripts/Tests/WDK.Tests.ps1

```diff
@@ -0,0 +1,14 @@
+Describe "WDK" {
+    BeforeAll {
+      $Global:WDKVersion = (Get-CimInstance -ClassName Win32_Product -Filter "Name = 'Windows Driver Kit'").Version
```

Comment:
Why don't we move to the  It "WDK exists" section and remove BeforeAll?

Reply:
According to AAA pattern https://medium.com/@pjbgf/title-testing-code-ocd-and-the-aaa-pattern-df453975ab80 `It` should not contain side-effect and initialisation code. It make sense if there few asserts on the same value.